src/test/scala-2.11/markets/participants/MarketParticipantLikeSpec.scala

```diff
@@ -16,18 +16,21 @@ limitations under the License.
 package markets.participants
 
 import akka.actor.ActorSystem
+import akka.agent.Agent
 import akka.testkit.{TestKit, TestProbe, TestActorRef}
 
 import java.util.UUID
 
 import markets.clearing.engines.BrokenMatchingEngine
+import markets.tickers.Tick
 import markets.{Filled, Remove, Add, MarketActor, Canceled, Accepted}
 import markets.orders.limit.LimitAskOrder
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560410985/issues/source?bid=2921361&fileBranchId=3019109#l27)